terraform.tfvars.example

```diff
@@ -71,6 +71,9 @@ logAnalyticsWorkspaceResourceGroupName = ""
 # The version of the AKS cluster.
 kubernetesVersion = "1.23.12"
 
+# Enable Key Vault purge protection
+keyVaultPurgeProtection = false
```

Comment:
In my opinion, we should set the default value to `true`, as this is the securer setting.